Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: concourse
-version: 9.0.0
-appVersion: 5.8.0
+version: 9.1.3
```

Comment:
Oh, you went to 9.1.3 because the latest on master before 6.0.0 was 9.1.2, right? (https://github.com/concourse/concourse-chart/commit/18e976753b38643f462699ae81d8f98547d0d2a3)

I think what we should do then have release/5.8.x based of that specific commit then, otherwise the release/5.8.x would be a bit far from the truth :thinking:  

wdyt? does that make sense?

Reply:
So I think the actual divergent point between 5.8.x and master is the commit 7df0973 that before you bump the master to appversion 6.0 at commit d4a07bafb7c089ad46695e4adf13cf4daed226e5

so i rebased master to 5.8.x first and drop all commits after 7df0973, then apply my commit for version bumping to 9.1.3.